### ISSUE TITLE
GPU is not available on Ubuntu 22

### ISSUE THREAD

**Issue by brusherru** [spacemeshos/gpu-post#83]
Originally reported here: https://github.com/spacemeshos/smapp/issues/1035
--
The user reported that neither internal Inter GPU nor Nvidia GPU is not available on his machine (running Ubuntu 22).
Previously on Smapp 0.2.8 (go-spacemesh 0.2.18-beta.0 and gpu-post 0.1.24), it works well.
So I assume we broke something in the newer versions.

**Comment by pigmej** [comment]
Ok located the problem.

```
nj@eolian:~/0.1.26$ ./gpu-setup-test --list
Available POST compute providers:
  1: [VULKAN] llvmpipe (LLVM 13.0.1, 256 bits)
  2: [CPU] CPU
nj@eolian:~/0.1.26$ cd ..
nj@eolian:~$ cd 0.1.27
nj@eolian:~/0.1.27$ ./gpu-setup-test --list
Available POST compute providers:
  1: [CPU] CPU
nj@eolian:~/0.1.27
```

on 0.1.27 it prints
```ERROR: loader_validate_instance_extensions: Extension VK_KHR_portability_enumeration not found in list of known instance extensions.```

**Comment by pigmej** [comment]
Should be resolved with 0.1.28 in #84
